doc/nrf/libraries/bluetooth_services/services/fast_pair.rst

```diff
@@ -30,9 +30,12 @@ The Fast Pair service also contains additional GATT characteristics under the fo
 Configuration
 *************
 
-Set the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option to enable the service.
+For projects with sysbuild support, the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option enables the service in the sysbuild configuration.
+The build system automatically sets the :kconfig:option:`CONFIG_BT_FAST_PAIR` Kconfig option in the main application image according to the ``SB_CONFIG_BT_FAST_PAIR`` Kconfig option configuration.
```

Comment:
Maybe we can rely on link to the user guide also here?

Reply:
We could explain that "special handling" is needed, because build system is used to generate the provisioning data.

Reply:
The integration guide uses different language (call to action) and outlines subsequent integration steps "You must do that", etc. The language used in the library documentation is more of a description

Reply:
Then maybe we could simply state here that: `:kconfig:option:`CONFIG_BT_FAST_PAIR` is used to enable the service and link to user guide for details on how to integrate Fast Pair and enable the option (with or without Sysbuild)?

Reply:
I try to avoid duplicating information